statchatexpress/models/userGame.js

```diff
@@ -15,14 +15,15 @@ export const UserGame = sequelize.define(
     GameName: {
       type: DataTypes.STRING,
       allowNull: false,
-      unique: false,
+      //user can only have one account per game:
+      unique: true,
```

Comment:
This will make it so the gameName is unique across **all rows**. If you remember last week when we were getting 500 errors when submitting the form - this is why. Could you fix this before merging?